docs/reference/issuers/acme/dns01.rst

```diff
@@ -33,6 +33,15 @@ it is also possible to have multiple instances of the same DNS provider on a
 single Issuer (e.g. two clouddns accounts could be set, each with their own
 name).
 
+Setting nameservers for DNS01 check
+===================================
+
+Cert-manager will check the correct DNS records exist before attempting a DNS01
+challenge.  By default, the DNS servers for this check will be taken from
+``/etc/resolv.conf``.  If this is not desired, the cert-manager controller
```

Comment:
Can you expand `If this is not desired,` to give some examples of where this may not be desired (i.e. if you have two authoritative nameservers for the same zone/are using split-brain/split-horizon DNS)?

Reply:
I've added a little extra, let me know if some larger examples would be needed